corehq/apps/hqadmin/static/hqadmin/js/visualizations.js

```diff
@@ -99,7 +99,8 @@ var HQVisualizations = function (options) {
             $(self.chart_tabs_id).on('click', 'a[data-toggle="hash-tab"]', function(){
                 $('.nvd3-chart').hide();
                 var $chart = $($(this).attr('href')).children('.nvd3-chart');
-                $chart.show();
+                $chart.show().removeClass('hidden');
```

Comment:
right, but more than just this b3 page uses this file. so I left the original functionality there since it doesn't break anything (it just doesn't apply)